types/@ember-decorators/utils/decorator.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '@ember-decorators/utils/decorator' {
+    export function decoratorWithParams(fn: (...params: any[]) => any): any;
+    export function decoratorWithRequiredParams(fn: (...params: any[]) => any): any;
+}
```

Comment:
Don't need this yet, I guess, but probably will at some point, so let's leave it in